packages/framer-motion/src/animation/legacy-popmotion/__tests__/inertia.test.ts

```diff
@@ -1,203 +0,0 @@
-import { syncDriver } from "./utils"
-import { inertia } from "../inertia"
-import { InertiaOptions } from "../../types"
-
-async function testInertia(
-    options: InertiaOptions,
-    expectation: number[],
-    resolve: () => void
-) {
-    const output: number[] = []
-
-    inertia({
-        keyframes: [0],
-        driver: syncDriver(200),
-        ...options,
-        onUpdate: (v) => output.push(Math.round(v)),
-        onComplete: () => {
-            expect(output).toEqual(expectation)
-            resolve()
-        },
-    })
-}
-
-describe("inertia", () => {
```

Comment:
Did these tests move somewhere else?

Reply:
Yeah `inertia()` used to be a function that ran its own animations so the tests are different (taken from Motion One) and now in `animation/generators/__tests__/inertia.ts`